index.js

```diff
@@ -2,20 +2,26 @@
 var argv = process.argv;
 
 module.exports = (function () {
+	var terminator = argv.indexOf('--');
+	var hasFlag = function (flag) {
+		var pos = process.argv.indexOf(flag);
+		return pos !== -1 && (terminator !== -1 ? pos < terminator : true);
+	};
+
 	if ('FORCE_COLOR' in process.env) {
 		return true;
 	}
 
-	if (argv.indexOf('--no-color') !== -1 ||
-		argv.indexOf('--no-colors') !== -1 ||
-		argv.indexOf('--color=false') !== -1) {
+	if (hasFlag('--no-color') ||
```

Comment:
Now sure what that would achieve? The terminator (`--`) is calculated before had and the individual arguments are found and compared against that position.